service_container/autowiring.rst

```diff
@@ -419,20 +459,33 @@ that alias:
         use App\Util\UppercaseTransformer;
         use App\Util\TransformerInterface;
         use App\Service\TwitterClient;
+        use App\Service\MastodonClient;
 
         // ...
         $container->autowire(Rot13Transformer::class);
         $container->autowire(UppercaseTransformer::class);
         $container->setAlias(TransformerInterface::class, Rot13Transformer::class);
+        $container->registerAliasForArgument(
+            TransformerInterface::class,
+            UppercaseTransformer::class,
+            'shoutyVariable'
```

Comment:
Before you wrote “$shoutyTransformer” do it need to match that variable here?

Reply:
Woops, good catch!